CommonNativeJarsTable.md

```diff
@@ -12,8 +12,8 @@ io.netty | [netty-transport-native-epoll](https://github.com/netty/netty) | yes
 io.netty | [netty-tcnative](https://github.com/netty/netty-tcnative) | yes | [yes](https://mvnrepository.com/artifact/io.netty/netty-tcnative) | 2.0.31
 org.fusesource.jansi | [jansi-native](https://github.com/fusesource/jansi-native) | yes | no |
 org.fusesource.leveldbjni | [leveldbjni-all](https://github.com/fusesource/leveldbjni) | no | no |
-org.fusesource.sigar | [sigar](https://github.com/hyperic/sigar) | no | no |
-org.apache.hadoop | [hadoop-lzo](https://github.com/twitter/hadoop-lzo) | no | no |
+org.fusesource.sigar | [sigar](https://github.com/hyperic/sigar) | yes (refer https://github.com/hyperic/sigar/pull/140) | no |
+org.apache.hadoop | [hadoop-lzo](https://github.com/twitter/hadoop-lzo) | yes | no |
 
 ---
 Updated on 2020-06-16
```

Comment:
pls update the date as well,  or just remove the data as git history will have the date

Reply:
There is a released version, not a date which typically follows after some time.

Reply:
@AGSaidi 
Thank you for confirmation.